### PR TITLE
influxctl 2.5.0

### PR DESCRIPTION
**Pull request by sanderson**
Adds `influxctl` 2.5.0 functionality to the Cloud Dedicated and Clustered docs.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md [hunk 1]
```diff
@@ -102,3 +114,20 @@ influxctl database create \
   --max-columns 150 \
   mydb
 ```
+
+### Create a database with with a custom partition template
+
+The following example creates a new `mydb` database and applies a partition
+template that partitions by two tags (`room` and `sensor-type`) and by week using
+the time format `%Y wk:%W`:
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:122]: In define-custom-partitions you show `wk:%U`, and `wk:%W`, I assume either work as my understanding is the difference is when the day of the week starts, but do we want to be consistent?

Same question below

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/table/_index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  InfluxDB Cloud Dedicated cluster.
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl
+weight: 201
+cascade:
+  metadata: [influxctl 2.5.0+]
+---
+
+The `influx table` command and its subcommands manage tables in an
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/table/_index.md:14]:
```suggestion
The `influxctl table` command and its subcommands manage tables in an
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/table/_index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: influxctl table
+description: >
+  The `influx table` command and its subcommands manage tables in an
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/table/_index.md:4]:
```suggestion
  The `influxctl table` command and its subcommands manage tables in an
```

File: content/influxdb/clustered/reference/cli/influxctl/database/create.md [hunk 4]
```diff
@@ -78,6 +89,7 @@ _Also see [`influxctl` global flags](/influxdb/clustered/reference/cli/influxctl
 - [Create a database with an infinite retention period](#create-a-database-with-an-infinite-retention-period)
 - [Create a database with a 30-day retention period](#create-a-database-with-a-30-day-retention-period)
 - [Create a database with non-default table and column limits](#create-a-database-with-non-default-table-and-column-limits)
+- [Create a database with with a custom partition template](#create-a-database-with-with-a-custom-partition-template)
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/database/create.md:92]:
```suggestion
- [Create a database with a custom partition template](#create-a-database-with-a-custom-partition-template)
```

File: content/influxdb/clustered/reference/cli/influxctl/database/create.md [hunk 5]
```diff
@@ -101,3 +113,20 @@ influxctl database create \
   --max-columns 150 \
   mydb
 ```
+
+### Create a database with with a custom partition template
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/database/create.md:117]:
```suggestion
### Create a database with a custom partition template
```

File: content/influxdb/clustered/reference/cli/influxctl/table/_index.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+---
+title: influxctl table
+description: >
+  The `influx table` command and its subcommands manage tables in an InfluxDB cluster.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/table/_index.md:4]:
```suggestion
  The `influxctl table` command and its subcommands manage tables in an InfluxDB cluster.
```

File: content/influxdb/clustered/reference/cli/influxctl/table/_index.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+menu:
+  influxdb_clustered:
+    parent: influxctl
+weight: 201
+cascade:
+  metadata: [influxctl 2.5.0+]
+---
+
+The `influx table` command and its subcommands manage tables in an InfluxDB cluster.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/table/_index.md:13]:
```suggestion
The `influxctl table` command and its subcommands manage tables in an InfluxDB cluster.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md [hunk 8]
```diff
@@ -9,10 +9,11 @@ menu:
 weight: 301
 related:
   - /influxdb/cloud-dedicated/admin/custom-partitions/define-custom-partitions/
+  - /influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/
 ---
 
 The `influxctl database create` command creates a new database with a specified
-retention period in an InfluxDB Cloud Dedicated cluster.
+retention period in an {{< product-name omit=" Clustered" >}} cluster.
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:16]: Should this be `a`? "a clustered cluster"? Not quite sure how that gets resolved. This is in a few places.

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Just a few c-p errors.

**Review by powersj — COMMENTED** [review]
Thank you for putting this together so quickly. I have two questions below.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md (hunk 8 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:16]: The product-name shortcode itself would resolve to InfluxDB Clustered, but "an InfluxDB Clustered cluster" doesn't read well, so we omit " Clustered" from the product name and it resolves as "an InfluxDB cluster."

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:122]: Nice catch. I'll update.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md (hunk 8 above)
Comment (powersj) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md:16]: aaah I read right past the "omit" part. Thanks!

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**